contracts/script/README.md

```diff
@@ -30,6 +30,8 @@ The magical deploy order is as follows:
 12. [FirstUsdtDepositFromL1ToL2.s.sol](./configure/FirstUsdtDepositFromL1ToL2.s.sol)
 13. [SetMM.s.sol](./configure/SetMM.s.sol)
 14. [AllowRouterToTransfer.s.sol](./configure/AllowRouterToTransfer.s.sol)
+15. [DeployPR1T1XChainReader.s.sol](./deploy/DeployPR1T1XChainReader.s.sol)
+16. [DeployL2T1XChainReader.s.sol](./deploy/DeployL2T1XChainReader.s.sol)
 
 ## Deploy 7683 Contract
```

Comment:
Shall we add `DeployT1ERC7683Pull` to this section?

Reply:
done